resources/migrations/001_update_migrations.yaml

```diff
@@ -6929,6 +6929,30 @@ databaseChangeLog:
             constraintName: fk_dashboard_collection_id
             onDelete: CASCADE
 
+  - changeSet:
+      id: v50.2024-05-03T16:29:53
+      author: johnswanson
+      comment: Create the Trash collection
+      changes:
+        - sql:
+            sql: >-
+              INSERT INTO collection (name, slug, entity_id, type) VALUES ('Trash', 'trash', 'trashtrashtrashtrasht', 'trash');
+              INSERT INTO permissions (object, group_id)
```

Comment:
This made me wonder — do we ensure that the trash collection's permissions can't be modified via the API?

Reply:
Yep, we do: https://github.com/metabase/metabase/blob/c6da3f9a5cc4c471511167b2520edeeaa92b0ac0/src/metabase/models/permissions.clj#L523-L524